Dockerfile

```diff
@@ -156,7 +156,7 @@ RUN curl -kSs https://pyenv.run | bash \
 
 RUN ARCH=$(arch | sed s/aarch64/arm64/) \
     && if [ "$ARCH" == "arm64" ]; then \
-    pip install -U scancode-toolkit-mini==$SCANCODE_VERSION; \
+    pip install -U scancode-toolkit-mini==$SCANCODE_VERSION licensedcode-data setuptools==$PYTHON_SETUPTOOLS_VERSION; \
```

Comment:
Please explain a few more details in the commit message. I *assume* the background to be that `scancode-toolkit-mini` (emphasis on *mini*) does not ship with the `scancode-license-data` command, so you install `licensedcode-data` for it? And what is `setuptools` needed for?

Reply:
> There's some missing dependencies.

This should definitely go to the commit message as the explanation.